recipe/meta.yaml

```
@@ -1,66 +1,73 @@
 {% set name = "tsfresh" %}
-{% set version = "0.18.0" %}
-{% set sha256 = "ebfc1505bc0c0148909a5009190ff49f8ffe4c8c661f7cafa9da6a6ae1a9abf4" %}
+{% set version = "0.21.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9942c2f78523ea12d0403b443293e45d4f1e78f750e5264fabddd2575cd75c43
 
 build:
   number: 0
-  # 2021/10/6: Skip building on osx-arm64 because statsmodels>=0.9.0 isn't available on this platform yet.
-  skip: true  # [arm64]
-  noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+  skip: true  # [py<310]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vvv
   entry_points:
     - run_tsfresh = tsfresh.scripts.run_tsfresh:main
 
 requirements:
   host:
-    - python >=3.5
     - pip
-    - setuptools_scm
+    - python
     - setuptools
+    - setuptools_scm
+    - pyscaffold >=3.3a0,<4
   run:
-    - python >=3.5
+    - python
     - requests >=2.9.1
-    - numpy >=1.15.1
+    - numpy {{ numpy }}
```

```suggestion
    - numpy  >=1.15.1
```
It's only a runtime dependency https://github.com/blue-yonder/tsfresh/blob/v0.21.0/setup.cfg#L38C10-L38C19. We use `numpy {{ numpy }}` only in `host` if the package has C API